clientv3/integration/cluster_test.go

```diff
@@ -186,4 +206,14 @@ func TestMemberAddForLearner(t *testing.T) {
 	if len(learners) != 1 {
 		t.Errorf("Added 1 learner node to cluster, got %d", len(learners))
 	}
+	capi.MemberPromote(context.Background(), resp.Member.ID)
```

Comment:
I wonder what happens to this test when we add progress check for learner.
1) since we only add learner to cluster but not actually launch the learner `etcd` process in this test, leader cannot get progress on a "unstarted" member?
2) if we decided to only send promote request to leader, capi (as a client taking to random sever in cluster) might not work.

Maybe add this integration test once the progress check is finalized and added?

Reply:
make sense